tests/base.py

```diff
@@ -0,0 +1,22 @@
+import json
+import os
+import random
+import string
+
+from os.path import expanduser
+
+
+FILE = 'token.json'
+AUTH_TOKEN_PATH = "{}/.evalai/".format(expanduser('~'))
```

Comment:
We need to make a list of things that need to go in a common config file.

Reply:
Okay, I'll start an issue.

Reply:
@varunagrawal can you check #37, I've added this. :)